src/app/pages/jails/jail-add/jail-add.component.ts

```diff
@@ -118,538 +127,644 @@ export class JailAddComponent implements OnInit {
+                 Use <b>sysvmsg</b>, <b>sysvsem</b>, and <b>sysvshm</b> \
+                 instead.'),
+    },
+    {
       type: 'checkbox',
       name: 'allow_raw_sockets',
-      placeholder: T('allow_raw_sockets'),
-      tooltip: T('Select this to allow utilities like <a\
- href="https://www.freebsd.org/cgi/man.cgi?query=ping&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
- target="_blank">ping(8)</a> and <a\
- href="https://www.freebsd.org/cgi/man.cgi?query=traceroute&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
- target="_blank">traceroute(8)</a> to operate inside the jail. When\
- checked, the source IP addresses are enforced to comply with the IP\
- address bound to the jail, ignoring the the IP_HDRINCL flag on the\
- socket.'),
-    }, {
+      placeholder: T('allow.raw_sockets'),
+      tooltip: T('Set to allow utilities like <a \
+href="https://www.freebsd.org/cgi/man.cgi?query=ping&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
+target="_blank">ping(8)</a> and <a \
+href="https://www.freebsd.org/cgi/man.cgi?query=traceroute&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
+target="_blank">traceroute(8)</a> to operate inside the jail. When \
+                 checked, source IP addresses are enforced to comply \
+                 with the IP addresses bound to the jail, ignoring the \
+                 IP_HDRINCL flag on the socket.'),
+    },
+    {
       type: 'checkbox',
       name: 'allow_chflags',
-      placeholder: T('allow_chflags'),
-      tooltip: T('Check this to treat jail users as privileged and allowed\
- to manipulate system file flags subject to the usual constraints on\
- <b>kern.securelevel</b>.'),
-    }, {
+      placeholder: T('allow.chflags'),
+      tooltip: T('Set to treat jail users as privileged and allow the \
+                 manipulation of system file flags. <b>securelevel</b> \
+                 constraints are still enforced.'),
+    },
+    {
       type: 'checkbox',
       name: 'allow_mount',
-      placeholder: T('allow_mount'),
-      tooltip: T('Check to allow privileged users inside the jail to mount\
- and unmount filesystem types marked as jail-friendly.'),
-    }, {
+      placeholder: T('allow.mount'),
+      tooltip: T('Set to allow privileged users inside the jail to mount \
+                 and unmount filesystem types marked as jail-friendly.'),
+    },
+    {
       type: 'checkbox',
       name: 'allow_mount_devfs',
-      placeholder: T('allow_mount_devfs'),
-      tooltip: T('Check to allow privileged users inside the jail to mount\
- and unmount the devfs file system. This permission is effective only\
- together with <b>allow_mount</b> and if <b>enforce_statfs</b> is set to\
- a value lower than <i>2</i>.'),
-    }, {
+      placeholder: T('allow.mount.devfs'),
+      tooltip: T('Set to allow privileged users inside the jail to mount \
+                 and unmount the devfs file system. This permission is \
+                 only effective when <b>allow_mount</b> and \
```

Comment:
allow_mount in the UI is a checkbox and can't be set to a numeric value.

Reply:
fixed